lib/series.js

```diff
@@ -0,0 +1,166 @@
+var assert = require('assert');
+var _ = require('lodash');
+
+
+/** Types supported for reporting */
+var types = {
+  String: function(v) { return /string/.test(typeof(v)); },
+  Number: function(v) { return /number/.test(typeof(v)); },
+  Any:    function(v) { return /string|number/.test(typeof(v)); }
+};
+
+// Export types
```

Comment:
Good point... I have too many stupid comments like that...

Note, this is not under babeljs, we should figure out how to do libs correctly with babeljs. I suspect compile on publish...